frontend/src/component/feature/CreateFeature/useFlagLimits.test.ts

```diff
@@ -0,0 +1,71 @@
+import { renderHook } from '@testing-library/react';
+import { useFlagLimits } from './CreateFeature';
+import { vi } from 'vitest';
+
+vi.mock('hooks/useUiFlag', async (importOriginal) => {
```

Comment:
this breaks the established pattern where we use msw for mocking HTTP

Reply:
Sure! I can see if just mocking the endpoint will do the trick.

Reply:
Actually, I've spent like 90 minutes now trying to get this to work as expected, so I'm gonna leave it in for now. We do use this same pattern two other places in the code. Because it's only a temporary requirement and because it's in tests, I don't want to spend more time on it.

If you have a solution for it that's easy to implement, I'm happy to have a look at that and set it up in a follow-up PR, though.